lib/talkable/api/share.rb

```diff
@@ -1,18 +1,40 @@
+# More at http://docs.talkable.com/api_v2/shares.html
 module Talkable
   module API
     class Share < Base
-      CHANNEL_FACEBOOK          = "facebook".freeze
-      CHANNEL_FACEBOOK_MESSAGE  = "facebook_message".freeze
-      CHANNEL_TWITTER           = "twitter".freeze
-      CHANNEL_SMS               = "sms".freeze
-      CHANNEL_OTHER             = "other".freeze
+      # social
+      VIA_FACEBOOK    = "facebook"
+      VIA_TWITTER     = "twitter"
+      VIA_LINKEDIN    = "linkedin"
+      VIA_WHATSAPP    = "twitter"
+      VIA_SMS         = "sms"
+      VIA_OTHER       = "other"
+
+      # direct
+      SEND_EMAIL      = "email"
+      SEND_FB_MESSAGE = "facebook_message"
```

Comment:
`facebook_message` is supposed to be direct but since the message is sent to unknown for Talkable user, `facebook_message` is social